mantisshrimp/datasets/fridge/transforms.py

```diff
@@ -0,0 +1,61 @@
+__all__ = ["train_albumentations_tfms_fridge", "valid_albumentations_tfms_fridge"]
+
+from mantisshrimp.imports import *
+from mantisshrimp.core import *
+from mantisshrimp.transforms.albu_transform import *
+from mantisshrimp.utils.utils import *
+
+
+def train_albumentations_tfms_fridge():
+    """ Composes a Pipeline of Albumentations Transforms for Fridge Objects dataset at the train stage
+
+    Returns:
+        AlbumentationsTransform: Pipeline of Albumentations Transforms
+    
+    
+    Examples::
+        >>> train_tfms = train_albumentations_tfms_fridge(train=True)
+        >>> train_ds = Dataset(train_records, train_tfms)
+
+    [[https://albumentations.readthedocs.io/en/latest/_modules/albumentations/augmentations/transforms.html/|Albumentations Transforms ]]
+    """
+    import albumentations as A
+
+    # ImageNet stats
+    imagenet_mean, imagenet_std = IMAGENET_STATS
+
+    return AlbuTransform(
+        [
+            A.LongestMaxSize(384),
+            A.RandomSizedBBoxSafeCrop(320, 320, p=0.3),
```

Comment:
I believe LongestMaxSize will not work, because efficient-det needs a fixed squared input that also needs to be divisible by 128